Install development interfaces and generate their paths in the upstream build.
Keep runtime libraries in the public lib directory.

--- a/src/liblzma/liblzma.pc.in
+++ b/src/liblzma/liblzma.pc.in
@@ -1,10 +1,10 @@
 # SPDX-License-Identifier: 0BSD
 # Author: Lasse Collin
 
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
+prefix=${pcfiledir}/../..
+exec_prefix=${prefix}
+libdir=${prefix}/lib
+includedir=${prefix}/include/liblzma-5.8.3
 
 Name: liblzma
 Description: General purpose data compression library
@@ -13,5 +13,5 @@
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}
 Cflags.private: -DLZMA_API_STATIC
-Libs: -L${libdir} -llzma
+Libs: ${libdir}/liblzma.so
 Libs.private: @PTHREAD_CFLAGS@ @LIBS@
